README.md

```diff
@@ -264,89 +299,93 @@ You can find our short- to midterm feature plans in the [public roadmap](https:/
+| [config](https://github.com/code-kern-ai/refinery-config) | Configuration of _refinery_. Amongst others, this manages endpoints and available language models for spaCy. |
+| [tokenizer](https://github.com/code-kern-ai/refinery-tokenizer) | Tokenizer for _refinery_. Manages the creation and storage of spaCy tokens for text-based record attributes and supports multiple language models. |
+| [gateway](https://github.com/code-kern-ai/refinery-gateway) | Gateway for _refinery_. Manages incoming requests and holds the workflow logic. To interact with the gateway, the UI or Python SDK can be used. |
+| [authorizer](https://github.com/code-kern-ai/refinery-authorizer) | Evaluates whether a user has access to certain resources. |
+| [websocket](https://github.com/code-kern-ai/refinery-websocket) | Websocket module for refinery. Enables asynchronous notifications inside the application. |
+| [lf-exec-env](https://github.com/code-kern-ai/refinery-lf-exec-env) | Execution environment for labeling functions. Containerized function as a service to execute user-defined Python scripts. |
+| [ac-exec-env](https://github.com/code-kern-ai/refinery-ac-exec-env) | Execution environment for attribute calulaction. Containerized function as a service to generate new attributes using Python scripts. |
+| [updater](https://github.com/code-kern-ai/refinery-updater) | Updater for _refinery_. Manages migration logic to new versions if required. |
+| [neural-search](https://github.com/code-kern-ai/refinery-neural-search) | Neural search for _refinery_. Manages similarity search powered by Qdrant and outlier detection, both based on vector representations of the project records. |
+| [zero-shot](https://github.com/code-kern-ai/refinery-zero-shot) | Zero-shot module for _refinery_. Enables the integration of 🤗 Hugging Face zero-shot classifiers as an off-the-shelf no-code heuristic. |
+| [entry](https://github.com/code-kern-ai/refinery-entry) | Login and registration screen for refinery. Implemented via Ory Kratos. |
+| [ui](https://github.com/code-kern-ai/refinery-ui) | UI for _refinery_. Used to interact with the whole system; to find out how to best work with the system, check out our docs. |
+| [doc-ock](https://github.com/code-kern-ai/refinery-doc-ock) | Usage statistics collection for _refinery_. If users allow it, this collects product insight data used to optimize the user experience. |
+| [gateway-proxy](https://github.com/code-kern-ai/refinery-gateway-proxy) | Gateway proxy for _refinery_. Manages incoming requests and forwards them to the gateway. Used by the Python SDK. |
+| [parent-images](https://github.com/code-kern-ai/refinery-parent-images/tree/parent-image) | Shared images used by _refinery_. Used to reduce the required space for _refinery_. _Not yet listed in architecture diagram_ |
```

Comment:
link doesnt work

Reply:
also i think this is outdated since we now use a submodule & n parent image repos e.g. https://github.com/code-kern-ai/refinery-exec-env-parent-image

Reply:
fixed that